oe_theme.theme

```diff
@@ -1321,13 +1321,20 @@ function _oe_theme_trim_textfield($text, int $size = 150) {
  */
 function oe_theme_preprocess_paragraph__oe_facts_figures(array &$variables): void {
   $paragraph = $variables['paragraph'];
-  $variables['title'] = $paragraph->get('field_oe_title')->value;
-  $variables['view_all']['path'] = $paragraph->get('field_oe_link')->first()->getUrl();
-  $variables['view_all']['label'] = $paragraph->get('field_oe_link')->first()->get('title')->getValue();
+  if (!empty($paragraph->get('field_oe_title')->value)) {
```

Comment:
no need to check here as the field is required.

Reply:
We are still checking if this fill is empty.